### PR TITLE
User side information in gcmc

### PR DESCRIPTION
**Pull request by nishiba**

### CODE REVIEW

**Comment by nishiba** [comment]
@hirosassa @6syun9 Please check my code. Thanks.

**Comment by 6syun9** [comment]
LGTM 💯

File: redshells/train/train_graph_convolutional_matrix_completion.py [hunk 1]
```diff
@@ -59,7 +66,13 @@ def run(self):
         ratings = df[self.rating_column_name].values
 
         model = GraphConvolutionalMatrixCompletion(
-            user_ids=user_ids, item_ids=item_ids, ratings=ratings, item_features=item_features, **self.model_kwargs)
+            user_ids=user_ids,
+            item_ids=item_ids,
+            ratings=ratings,
+            user_features=user_features,
+            item_features=item_features,
+            test_size=0.2,  # TODO
```
Comment (hirosassa) [inline redshells/train/train_graph_convolutional_matrix_completion.py]: "todo" is remaining. I think it is OK to hand over to next improvement

Comment (nishiba) [reply redshells/train/train_graph_convolutional_matrix_completion.py]: I use class member variable for `test_size`. Thanks!

**Pull request merged**